src/couch/test/couchdb_os_daemons_tests.erl

```diff
@@ -56,9 +56,16 @@ setup(DName) ->
     {Ctx, OsDPid}.
 
 teardown(_, {Ctx, OsDPid}) ->
-    test_util:stop_sync_throw(OsDPid, fun() ->
-        exit(OsDPid, shutdown)
-    end, {timeout, os_daemon_stop}, ?TIMEOUT),
+    try
+        test_util:stop_sync_throw(OsDPid, fun() ->
+            exit(OsDPid, shutdown)
+        end, {timeout, os_daemon_stop}, ?TIMEOUT)
+    catch
+        {timeout, os_daemon_stop} ->
+            Msg = "WARNING: OS daemons test stop ~p msec timeout exceeded",
+            io:format(standard_error, Msg, [?TIMEOUT]),
```

Comment:
I suspect it has something to do with how rebar sets `DEBUG` - `NODEBUG` macros, but not sure, maybe _it is_ how travis treats stdout and stderr.